card/src/main/java/com/adyen/checkout/card/CardComponentState.kt

```diff
@@ -14,11 +14,11 @@ import com.adyen.checkout.components.core.paymentmethod.CardPaymentMethod
 /**
  * Represents the state of [CardComponent].
  */
-data class CardComponentState(
+open class CardComponentState(
```

Comment:
There seems to be no `BcmcComponentState`. I think it would be nice to create one, instead of using `CardComponentState` for the `BcmcComponent`